Specs/Types/Date.js

```diff
@@ -424,6 +424,9 @@ describe('Date', function(){
 		it('should return the minutes as two digits', function(){
 			expect(d.format('%M')).toEqual('02');
 		});
+		it('should return the month as one digit', function(){
+			expect(d.format('%n')).toEqual('11');
```

Comment:
I should get some more coffee before my first commits on sundays... ;-)
Thx. Fixed.